package.json

```diff
@@ -24,8 +24,8 @@
   "scripts": {
     "postinstall": "./bin/patch_webdriver.sh",
     "lint": "eslint .",
-    "build": "./bin/make_xpi.sh",
-    "test": "export XPI_NAME=extension/share-button-study.xpi && npm run build && mocha test --retry 2",
+    "build": "./bin/make_xpi.sh > /dev/null 2>&1",
+    "test": "export XPI_NAME=extension/share-button-study.xpi && npm run build && mocha test/button_test.js --retry 2",
```

Comment:
This is in order to silence the output of `make_xpi.sh` for the test harness, but maybe it makes more sense to move this to the `test_harness.js` file instead?